From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tobias Christiansen <tobyase@serenityos.org>
Date: Thu, 24 Mar 2022 15:41:48 +0100
Subject: [PATCH] Tell prng.c that we don't have arc4random_buf

FIXME: This function does exist, perhaps outdated or some issue -
       explain the issue here if so.
---
 src/prng.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/prng.c b/src/prng.c
index 004906d..d73b05c 100644
--- a/src/prng.c
+++ b/src/prng.c
@@ -29,6 +29,7 @@
 #  undef HAVE_arc4random_buf
 #endif
 
+#undef HAVE_arc4random_buf
 #ifdef HAVE_arc4random_buf
 #  define HAVE_ARC4RANDOM
 #  ifndef ARC4RANDOM_IN_BSD
